src/Qowaiv/Qowaiv.csproj

```diff
@@ -5,8 +5,14 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net5.0;net6.0;net7.0</TargetFrameworks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <Version>6.4.0</Version>
+    <Version>6.4.1</Version>
     <PackageReleaseNotes>
+v6.4.1
+- Clock.SetTimeForCurrentContext supports async. #304
+- Fix issue with not closed comment in parsing email adresses. #298
```

Comment:
Are all these changes in this PR, or are they previous changes that didn't get a version bump yet?

Reply:
Yep. They where considered too minor to get a separate release.